api/db-cdr/csv/survey_question_map.csv

```diff
@@ -0,0 +1,82 @@
+id,survey_concept_id,question_concept_id,path,is_main
```

Comment:
How is this file generated and how did you determine the schema for this? Please document somewhere.

Reply:
I have made this file manually by looking at the survey pdf (and the concept code of the questions). There is only one survey (the basics) which has only one question which has sub questions.

Reply:
I have outlined the description in the current readme. LEt me know if it is understood. It seems okay to me.